backtrace-library/src/main/java/backtraceio/library/models/attributes/ReportDataBuilder.java

```diff
@@ -0,0 +1,54 @@
+package backtraceio.library.models.attributes;
+
+import java.util.Map;
+
+import backtraceio.library.common.TypeHelper;
+
+public class ReportDataBuilder {
+
+    /**
+     * Divide custom user attributes into primitive and complex attributes and add to this object. By default nullable values will be included.
+     *
+     * @param attributes client's attributes
+     * @return Report data attributes divided into attributes and annotations
+     */
+    public static ReportDataAttributes getReportAttributes(Map<String, Object> attributes) {
+        return getReportAttributes(attributes, false);
+    }
+
+    /**
+     * Divide custom user attributes into primitive and complex attributes and add to this object
+     *
+     * @param attributes   client's attributes
+     * @param skipNullable define attributes behavior on nullable value. By default all nullable attributes
+     *                     will be included in the report. For some features like metrics, we don't want to send
+     *                     nullable values, because they can generate invalid behavior/incorrect information.
+     * @return Report data attributes divided into attributes and annotations
+     */
+    public static ReportDataAttributes getReportAttributes(Map<String, Object> attributes, boolean skipNullable) {
```

Comment:
I think you meant `skipNull`. `Nullable` would be a characteristic of variable, i.e. it can be set to null.

Reply:
Oh right

Reply:
can you also change `nullable` to `null` in the param doc?